compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -129,6 +152,18 @@ printTestCoverage dalfs results
                                                      }) <- [SS.nodeNode n]
         , Just templateId <- [node_ExerciseTemplateId]
         ]
+    missingChoices =
+        [ (LF.moduleNameString $ LF.moduleName m) <> ":" <>
+        (T.concat $ LF.unTypeConName $ LF.tplTypeCon t) <>
```

Comment:
Instead of mixing the template and the choice into a single string, can we just work with a tuple? For templates, the string makes sort of sense since we get that from the scenario protobuf but choice are separate. No need to turn everything into a single string.

Reply:
Totally agreed.